policybot/pkg/storage/spanner/read.go

```diff
@@ -204,6 +204,22 @@ func (s store) ReadBotActivity(context context.Context, orgLogin string, repoNam
 	return &result, nil
 }
 
+func (s store) ReadTestResultByName(context context.Context, orgLogin string,
```

Comment:
I've updated the naming conventions of these methods, to be  consistent please drop the ByName suffix.